api/nodes/serializers.py

```diff
@@ -373,3 +379,29 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
```

Comment:
Thinking about the json-api spec for relationships links.  I'm wondering if we should include links in addition to the data in the response. See http://jsonapi.org/format/#fetching-relationships-responses-200
```
{
  "links": {
    "self": "/nodes/abcde/relationships/institution",
    "related": "/nodes/abcde/institution"
  },
  "data": {
    "type": "institution",
    "id": "UVA"
  }
}
```